apps/react-native/package.json

```diff
@@ -31,8 +35,6 @@
     "@react-native/typescript-config": "0.73.1",
```

Comment:
🟡 Is `@react-native/typescript-config` and `@react-native/eslint-config` still needed?

Reply:
good catch 👍